iModelCore/ECDb/ECDb/ECSql/ECSqlParser.cpp

```diff
@@ -22,6 +23,18 @@ std::unique_ptr<Exp> ECSqlParser::Parse(ECDbCR ecdb, Utf8CP ecsql, IssueDataSour
 
     if (parseTree == nullptr || !error.empty()) {
         Issues().ReportV(IssueSeverity::Error, IssueCategory::BusinessProperties, IssueType::ECSQL, "Failed to parse ECSQL '%s': %s", ecsql, error.c_str());
+
+        // Check if Min/Max function with multiple args was called. 
+        std::cmatch maxRegexMatches;
+        std::regex_search(ecsql, maxRegexMatches, std::regex(R"rx([(=,\s]MAX\(\d+,.*\))rx", std::regex_constants::icase));

```

Comment:
What about if I do `SELECT 'MAX(2,3)' AS str`  ? Or a comment in ECSQL container similar string? Also if ecsql look like this
```sql
select MAX
( 3, 
4)
FROM test.Goo
```

The solution is flawed. 

Use parse tree to find how many parameter MAX or MIN have.

Reply:
The Min/Max functions have a grammar rule defined only for single args. So, if I call them with multiple args, the parse tree generation itself fails giving a syntax error.
That's why I had to resort to regex.

Reply:
You can remove rules for Max/Min from grammar. It will be then detected as regular function with n param.